docs/ca/dev/configure-data-import-from-an-s3-bucket.md

```diff
@@ -17,620 +17,59 @@ This document describes how to replace the original CsvReader with the one based
-            $this->calculateCount();
-        }
-
-        return $this->count;
-    }
-
-    /**
-     * @return void
-     */
-    protected function calculateCount()
-    {
-        $this->count = 0;
-
-        $this->rewind();
-
-        while (($row = $this->getRowAndGoNext()) !== null) {
-            if ($this->isEmpty($row)) {
-                continue;
-            }
-
-            $this->count++;
-        }
-
-        $this->rewind();
-    }
+* Install the required modules
 
-    /**
-     * @param array $row
-     *
-     * @return bool
-     */
-    protected function isEmpty(array $row): bool
-    {
-        if (count($row) == 1 && $row[0] == '') {
-            return true;
-        }
-
-        return false;
-    }
-
-    /**
-     * @return int
-     */
-    public function key(): int
-    {
-        return $this->key;
-    }
-
-    /**
-     * @return void
-     */
-    protected function incrementKey()
-    {
-        $this->key++;
-    }
-
-    /**
-     * @return void
-     */
-    protected function incrementImportableKey()
-    {
-        $this->importableKey++;
-    }
-
-    /**
-     * @return void
-     */
-    protected function resetKeys()
-    {
-        $this->key = 0;
-        $this->importableKey = 0;
-    }
-
-    /**
-     * @return bool
-     */
-    public function valid(): bool
-    {
-        if ($this->limit !== null && $this->limit !== 0) {
-            if ($this->offset !== null) {
-                return ($this->key() < $this->offset + $this->limit);
-            }
-        }
-
-        return $this->importableKey < $this->count;
-    }
-
-    /**
-     * @return void
-     */
-    public function rewind()
-    {
-        rewind($this->fileObject);
-        $this->resetKeys();
-
-        if ($this->offset) {
-            fseek($this->fileObject, $this->offset - 1);
-        }
-
-        if ($this->csvReaderConfiguration->hasHeader()) {
-            $this->next();
-        }
-    }
-}
-```
-
-</details>
-
-4. Add the `addFlysystemService()` method and the `SERVICE_FLYSYSTEM` constant:
-
-**src/Pyz/Zed/DataImport/DataImportDependencyProvider.php**
-```php
-...
-use Spryker\Service\Flysystem\FlysystemServiceInterface;
-...
-
-class DataImportDependencyProvider extends SprykerDataImportDependencyProvider
-{
-    ...
-    public const SERVICE_FLYSYSTEM = 'SERVICE_FLYSYSTEM';
-    ...
-
-    /**
-     * @param Container $container
-     *
-     * @return Container
-     */
-    private function addFlysystemService(Container $container): Container
-    {
-        $container[static::SERVICE_FLYSYSTEM] = function (Container $container): FlysystemServiceInterface {
-            return $container->getLocator()->flysystem()->service();
-        };
-
-        return $container;
-    }
-}
-```
-
-
-5. In `src/Pyz/Zed/DataImport/Business/DataImportBusinessFactory.php`, do the following:
+| NAME                      | VERSION |
+|---------------------------|---------|
+| DataImport                | ^1.25.0 |
```

Comment:
we should use the composer name - sometimes it is not obvious if you mean `spryker/my-module` or `spryker-shop/my-module`. Also its easier to copy&paste.

So suggestion is to use `spryker/data-import` and `spryker/flysystem-aws3v3-file-system`

Reply:
Fixed